src/app/settings/settings.component.ts

```diff
@@ -239,4 +247,36 @@ export class SettingsComponent extends RouterEnter {
     return true;
   }
 
+  async uploadProfileImage() {
+    const s3Config = this.filestackService.getS3Config('image');
+    const pickerOptions = {
```

Comment:
Do not need repeat all these, re-use filestack component, maybe just pass an additional parameter for the layout of the button